### PR TITLE
Probe info

### PR DESCRIPTION
**Pull request by ecomerford**
@chutten I was able to configure the fetch request so that it now maps the metric names to `metricOptions` state. However, I'm working on creating an `ItemAdapter` that will display the type and description, as well. 

I also moved the sample data into a data folder in the `src` directory. And imported the Autosuggest style file, but this made no changes to the component's style. Still working on that, as well!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,57 +5,39 @@ import {ViewSelector} from "./Components/ViewSelector.js";
...
 
+  componentWillMount = () => {
+    fetch('https://probeinfo.telemetry.mozilla.org/firefox/all/main/all_probes')
+    .then(response => {
+      return response.json();
+    }).then(data => {
+      var probeData = Object.values(data);
+      var metrics = probeData.map(item => item.name);
+      this.setState({probeInfo: probeData,})
+      this.setState({metricOptions: metrics,})
```
Comment (chutten) [inline src/App.js]: You should be able to combine these two calls:
```js
this.setState({
  probeInfo: probeData,
  metricOptions: metrics,
});
```

File: src/Components/MetricSelector.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import React from "react";
 import {Grid, Row} from "react-bootstrap";
-import Autosuggest from "react-bootstrap-autosuggest"
```
Comment (chutten) [inline src/Components/MetricSelector.js:3]: I recommend enabling a "semicolons are required" eslint rule

**Review by chutten — COMMENTED** [review]
In the built project are there styles that come from `Autosuggest.scss`? You should be able to inspect the autosuggest elements and see where their styles come from.

**Comment by chutten** [comment]
Travis is failing the build due to not being able to make a network request during tests. 

Not sure why it needs the network during tests, though...

**Comment by ecomerford** [comment]
@chutten I think the problem is to do with the fetch call, similar to this series of[ issues](https://github.com/facebook/create-react-app/issues/2251). I tried importing `fetch` as suggested [here](https://stackoverflow.com/questions/44700992/yarn-test-network-request-failed), but that did not fix the problem in my local tests. I guess the next step is to try installing [fetch-mock](http://www.wheresrhys.co.uk/fetch-mock/) or something similar, unless you have any other thoughts.

**Comment by chutten** [comment]
No ideas, sorry.

**Comment by ecomerford** [comment]
I'll work on installing `fetch-mock` then. Hopefully that helps! 

Regarding `Autosuggest.scss`: when I inspect the elements it appears that they are only inheriting style attributes from `App.css` and from bootstrap. Nothing from the `.scss` file that I imported.

Could be that the sass-loader isn't working properly? I'm not sure. I'm going to move on from that for now to focus on other things, but it would be nice to see the default style so that we can decide what changes we want to make to it.

**Comment by chutten** [comment]
I guess it means it is an unresolveable import. I'm a little surprised react's build step didn't complain it couldn't find it. I would expect `import 'file/what/doesnt/exist.scss'` would at least print a warning...

If fetch-mock gets the build working again, I say merge it so we can get more eyes on these commits. Maybe the wisdom of crowds will help out.

**Pull request merged**